templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -161,13 +173,15 @@ contents:
       # create bridge
       if ! nmcli connection show "$bridge_name" &> /dev/null; then
         ovs-vsctl --timeout=30 --if-exists del-br "$bridge_name"
-        add_nm_conn type ovs-bridge con-name "$bridge_name" conn.interface "$bridge_name" 802-3-ethernet.mtu ${iface_mtu}
+        add_nm_conn type ovs-bridge con-name "$bridge_name" conn.interface "$bridge_name" 802-3-ethernet.mtu ${iface_mtu} \
+        connection.autoconnect-slaves 1
       fi
 
       # find default port to add to bridge
       if ! nmcli connection show "$default_port_name" &> /dev/null; then
         ovs-vsctl --timeout=30 --if-exists del-port "$bridge_name" ${iface}
-        add_nm_conn type ovs-port conn.interface ${iface} master "$bridge_name" con-name "$default_port_name"
+        add_nm_conn type ovs-port conn.interface ${iface} master "$bridge_name" con-name "$default_port_name" \
```

Comment:
were we quoting `${iface}`?  Are we sure nobody's being clever and working on adding a way to put whitespace in interface names?

Reply:
There would be no harm in being extra cautious and quoting `iface` but that would be unrelated to this change as there are other places other than this were `iface` is not quoted.
It would not make sense of `iface` to have spaces as it is a device name.
A user could inject spaces into `iface` by manipulating the default interface hint file, which is unsupported anyway.